pcsx2/System.cpp

```diff
@@ -358,6 +358,18 @@ namespace HostMemoryMap {
 
 /// Attempts to find a spot near static variables for the main memory
 static VirtualMemoryManagerPtr makeMainMemoryManager() {
+	const uptr UpperBounds = 0U;
+	const bool Strict = true;
```

Comment:
I'd prefer if the fact that the value was `true` was clearly visible at the call site, not way at the top of the function.  This option is decided separately for each time we try to make a VMM, and isn't even the same for all three.

Reply:
If you want to consolidate things, make a
```cpp
auto tryMakeAt = [&memoryManagerName](uptr base) -> VirtualMemoryManagerPtr {
	std::make_shared<VirtualMemoryManager>(memoryManagerName, base, HostMemoryMap::Size, /*upper_bounds=*/0, /*strict=*/true);
	if (mgr->IsOk())
 		return mgr;
	return nullptr;
}
```
Then you can
```cpp
if (auto mgr = tryMakeAt(LegacyMemoryBase))
	return mgr;
```